modules/op-release-notes-1-8.adoc

```diff
@@ -152,25 +151,82 @@ Previously, to add the tasks and pipelines in the catalog to the database, you p
+* With this update, you can disable the login and rating features of your {tekton-hub} web console. To do so, omit adding a value for the `AUTH_BASE_URL` field in the `<tekton_hub_repository>/config/03-ui/30-ui-configmap.yaml` file.
 // https://github.com/tektoncd/hub/pull/542
 // Shiv Pratap Verma @pratap0007
 
-* With this update, you don't need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the catalog refresh API runs.
+* With this update, you don't need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the API starts running for the first time.
 // https://github.com/tektoncd/hub/pull/510
 // Shiv Pratap Verma @pratap0007
 
 * This update automatically refreshes the catalog every 30 minutes by calling the catalog refresh API job. This interval is user-configurable.
 // https://github.com/tektoncd/hub/pull/547
 // Shiv Pratap Verma @pratap0007
 
+[id="pac-new-features-1-8_{context}"]
+=== {pac}
+// {pac} owners: Shivam Mukhade Savita Ashture
+
+:FeatureName: {pac} (PAC)
+include::snippets/technology-preview.adoc[]
+
+* {pipelines-title} 1.7.x shipped with {pac} 0.5.x. The current update ships with {pac} 0.10.x. This change creates a new route in the `openshift-pipelines` namespace for the new controller. You must update this route in GitHub Apps or webhooks that use {pac}. To fetch the route, use the following command:
++
+[source,terminal]
+----
+oc get -n openshift-pipelines pipelines-as-code-controller  --template='https://{{ .spec.host }}'
+----
+// no link
+
+* With this update, {pac} supports GitLab integration with features such as the following:
+** ACL (Access Control List) on project or group
+** `/ok-to-test` support from allowed users
+** `/retest` support.
+//For more information, https://pkg.go.dev/github.com/openshift-pipelines/pipelines-as-code/pkg/provider/gitlab#section-readme
+// no link
+
+* With this update, you can perform advanced pipeline filtering with Common Expression Language (CEL). With CEL, you can match pipeline runs with different Git provider events through special annotations in the `PipelineRun` resource. For example:
++
+[source,yaml]
+----
+  ...
+  annotations:
+     pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && target_branch == "main" && source_branch == "wip"
+----
+// no link
+
+* With this update, {pac} supports multiple pipeline runs on the same event. The pipeline runs operate in parallel and report back to the Git provider interface.
+// no link
+
+* With this update, you can rerun a pipeline run by commenting `/test` or `/retest` on a pull request. You can also specify the pipeline run by name. For example, by entering `/test <pipelinerun_name>` or  `/retest <pipelinerun-name>`
```

Comment:
```suggestion
* With this update, you can re-run a pipeline run by commenting `/test` or `/retest` on a pull request. You can also specify the pipeline run by name. For example, by entering `/test <pipelinerun_name>` or  `/retest <pipelinerun-name>`
```
Might be better to rephrase so that you can avoid saying `re-run a ... run`

Reply:
Good point. Went with "you can test or retest..."